index.js

```diff
@@ -4,13 +4,16 @@
 module.exports = {
   name: 'ember-tether',
 
-  included: function(app) {
-    if (app.import) {
-      this.importBowerDependencies(app);
+  options: {
+    nodeAssets: {
+      tether: {
+        srcDir: 'dist',
+        import: ['js/tether.js']
+      }
     }
   },
 
-  importBowerDependencies: function(app) {
-    app.import(app.bowerDirectory + '/tether/dist/js/tether.js');
+  importBowerDependencies: function() {
```

Comment:
Is there any reason to keep this method around given that the call to it was removed?